src/components/Expander/Expander.jsx

```diff
@@ -2,179 +2,148 @@ import _ from 'lodash';
-		);
-
-		return (
-			<div
-				{...omitProps(passThroughs, Expander)}
-				className={cx(
-					'&',
-					{
-						'&-is-expanded': isExpanded,
-						'&-kind-highlighted': kind === 'highlighted',
-					},
-					className
-				)}
-				style={style}
-			>
-				<header className={cx('&-header')}>
-					<div className={cx('&-header-toggle')} onClick={this.handleToggle}>
-						<span className={cx('&-icon')}>
-							<ChevronIcon size={12} direction={isExpanded ? 'up' : 'down'} />
-						</span>
-						{labelChildProp && (
-							<span className={cx('&-text')}>{labelChildProp.children}</span>
-						)}
-					</div>
-					{additionalLabelContentChildProp && (
-						<div className={cx('&-additional-content')}>
-							{additionalLabelContentChildProp.children}
-						</div>
-					)}
-				</header>
-				<Collapsible
-					isExpanded={isExpanded}
-					rootType='section'
-					className={cx('&-content')}
-				>
-					{children}
-				</Collapsible>
-			</div>
-		);
-	},
-
-	handleToggle(event) {
-		this.props.onToggle(!this.props.isExpanded, {
-			event,
-			props: this.props,
-		});
-	},
-});
-
-export default buildHybridComponent(Expander);
-export { Expander as ExpanderDumb };
+	className: string`
+		Appended to the component-specific class names set on the root element.
+	`,
+
+	isExpanded: bool`
+		Indicates that the component is in the "expanded" state when true and in
+		the "unexpanded" state when false.
+	`,
+
+	onToggle: func`
+		Called when the user clicks on the component's header.  Signature:
+		\`(isExpanded, { event, props }) => {}\`
+	`,
+
+	style: object`
+		Passed through to the root element.
+	`,
+
+	Label: any`
+		Child element whose children represents content to be shown next to the
+		expander icon.
+	`,
+
+	AdditionalLabelContent: node`
+		Child element whose children respresent content to be shown inside
+		Expander.Label and to the right of it
+	`,
+
+	kind: oneOf(['simple', 'highlighted'])`
+		Renders different variants of Expander. 'simple' is default.
+		'highlighted' is more prominant.
+	`,
+};
+
+Expander.Label = () => null;
+Expander.Label.propName = 'Label';
+Expander.Label.displayName = 'Expander.Label';
+Expander.Label.propTypes = {
+	children: node`
+		Used to identify the purpose of this switch to the user -- can be any
+		renderable content.
+	`,
+};
+
+Expander.AdditionalLabelContent = () => null;
+Expander.AdditionalLabelContent.displayName = 'Expander.AdditionalLabelContent';
+Expander.AdditionalLabelContent.propName = 'AdditionalLabelContent';
+Expander.AdditionalLabelContent.propTypes = {
+	children: node`
+		Used to display additional information or/and actions next to expander label.
+	`,
+};
+
+//@todo maybe just let consumers compose the HoC directly so they can pass their own initialState and whatnot?
+//@todo we could keep the stateful component as the default.  Just need to tweak our module loader in Storybook
+const ExpanderStateful = withReducer(Expander, reducer, mapDispatchToProps, initialState);
```

Comment:
^^note

Reply:
The idea behind exporting the "hybrid component" as the default was because it allowed consumers to "take over" any of the props they needed without worrying too much. If that's a convention we break it's probably going to be a big change to get consumers to do the composition themselves. In a vacuum I'd agree that it's the right change to make though.

Reply:
yep the main difference here would be consumers either opt-in or opt-out of driving props controlled by the reducers.  With Hybrid Components consumers could use the stateful version but still drive the props themselves and the internal state would be ignored.  I'm not convinced the existing behavior is ideal since it leads to an amount of magic if you don't fully grok what is happening.  
 
The lift for consumers as it stands here would be searching for the 22 Hybrid Components where they also pass in a prop controlled by Hybrid state.